datafusion/physical-plan/src/aggregates/group_values/column_wise.rs

```diff
@@ -78,6 +79,38 @@ impl GroupValuesColumn {
             random_state: Default::default(),
         })
     }
+
+    /// Returns true if [`GroupValuesColumn`] supports for the specified schema
+    pub fn supported_schema(schema: &Schema) -> bool {
+        schema
+            .fields()
+            .iter()
+            .map(|f| f.data_type())
+            .all(Self::supported_type)
+    }
+
+    /// Returns true if the specified data type is supported by [`GroupValuesColumn`]
+    fn supported_type(data_type: &DataType) -> bool {
+        matches!(
+            *data_type,
+            DataType::Int8
```

Comment:
should we use `DataType::is_signed_integer` `DataType::is_unsigned_integer` 

we can also use `is_numeric` but it includes Decimals

Reply:
This list needs to be kept in sync with what special implementations are available in `GroupValuesColumns` -- I will add a comment to make that clearer